pymchelper/readers/shieldhit.py

```diff
@@ -252,6 +252,7 @@ def get_detector_unit(detector_type, geotyp):
             SHDetType.pet: ("/primary", "PET isotopes"),
             SHDetType.dletg: ("keV/um", "dose-averaged LET"),
             SHDetType.tletg: ("keV/um", "track-averaged LET"),
+            SHDetType.q: ("z^2/(MeV/um)", "Q factor"),
```

Comment:
stictly "z^2" is not a unit. How is charge number normally displayed? e-?
[ e-² amu / MeV ]

MeV/amu for the denominator would be better, since "um" may be mistaken with "micrometer"

Reply:
also, let's call it "beam quality Q" (as Armin did) instead of "Q-factor".

Reply:
and since amu is in MeV/c², Q transforms to [e²/c²]

So, we have two options:

[e²/c²] or [e² u / MeV]

Reply:
ok, I'll rename it. [e^2 / c^2] looks best for me.